manifest.json

```diff
@@ -1,4 +1,12 @@
 {
+  "cardanoconnect": {
+    "image": "hyperledger/firefly-cardanoconnect",
+    "tag": "main"
```

Comment:
This should be a release version, maybe we could release the cardano connector?

Reply:
yeah, It would be really helpful to have version numbers attached to the cardano image. Right now it's set up to just publish a new image every commit to main, but I could switch it to using actual releases.

Reply:
Yeah, I'll help get that going